covalent/_results_manager/result.py

```diff
@@ -282,13 +304,27 @@ def get_all_node_outputs(self) -> dict:
         Returns:
             node_outputs: A dictionary containing the output of every node execution.
         """
+        with Session(db.engine) as session:
 
-        return {
-            self._get_node_name(node_id): self._get_node_output(node_id)
-            for node_id in range(self._num_nodes)
-        }
+            lattice_id = (
+                session.query(models.Lattice)
+                .where(models.Lattice.dispatch_id == self.dispatch_id)
+                .first()
+                .id
+            )
+            electron_records = (
+                session.query(models.Electron)
+                .where(models.Electron.parent_lattice_id == lattice_id)
+                .all()
+            )
```

Comment:
potential tuple issue

Reply:
Turns out (1) is 1 and (1,) will be a tuple